### PR TITLE
fix(side-panel): remove unnecessary underline on hovering Backlog

### LINKED ISSUE #1878: Hyperlink of Blue color on "Backlog" and its count
Hyperlink of Blue color on "Backlog" and its count 

In invision designs the "Backlog" doesn't have a Hyperlink of blue color. (Small cosmetic issue).

Video link : https://drive.google.com/a/redhat.com/file/d/0B1qUq_laG6COTDVMNmFXRUtrRms/view?usp=sharing

### PR DESCRIPTION
**Pull request by debloper**
Closes #1878

### CODE REVIEW

**Comment by fabric8cd** [comment]
@debloper snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1962-fabric8-planner.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@debloper snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1962-fabric8-planner.int.rdu2c.fabric8.io

**Comment by nimishamukherjee** [comment]
@sanbornsen can you verify and approve this PR?

File: src/app/components/side-panel/side-panel.component.html [hunk 1]
```diff
@@ -1,9 +1,9 @@
-<a [routerLink]="[]" [queryParams]="{}">
+<div [routerLink]="[]" [queryParams]="{}">
```
Comment (sanbornsen) [inline src/app/components/side-panel/side-panel.component.html]: It has to be an anchor tag else you will not be able to right click on it and open it in a new tab.

Comment (debloper) [inline src/app/components/side-panel/side-panel.component.html]: Fair enough. CSS fix it is then, coming up.

**Comment by fabric8cd** [comment]
@debloper snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1962-fabric8-planner.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@debloper snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1962-fabric8-planner.int.rdu2c.fabric8.io

File: src/app/components/side-panel/side-panel.component.html (hunk 1 above)
Comment (sanbornsen) [inline src/app/components/side-panel/side-panel.component.html]: Shall we just close the PR then?

**Comment by fabric8cd** [comment]
@debloper snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1962-fabric8-planner.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@debloper snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1962-fabric8-planner.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@debloper snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1962-fabric8-planner.int.rdu2c.fabric8.io

**Pull request merged**